memote/suite/tests/test_basic.py

```diff
@@ -446,3 +418,53 @@ def test_find_unique_metabolites(read_only_model):
         total of {} ({:.2%}) unique metabolites in the model: {}""".format(
             len(ann["data"]), ann["metric"], truncate(ann["data"])))
     assert len(ann["data"]) < len(read_only_model.metabolites), ann["message"]
+
+
+@annotate(title="Number of Unique Metabolites", type="count")
+def test_find_duplicate_metabolites_in_compartments(read_only_model):
+    """
+    Expect there to be zero duplicate metabolites in the same compartments.
+
+    Metabolites may be transported into different compartments, and so models
```

Comment:
The main reason for having this test is to clean up merged models or models from automated reconstruction pipelines as these are prone to having identical metabolites from different namespaces (hence different IDs).

Please upgrade the description accordingly.